pulp_rpm/app/migrations/0045_modulemd_fields.py

```diff
@@ -15,8 +15,23 @@
 def populate_new_fields(apps, schema_editor):
     """Populate new fields of modulemd."""
     Modulemd = apps.get_model("rpm", "Modulemd")
+    ModulemdDefaults = apps.get_model("rpm", "ModulemdDefaults")
     modules_to_update = []
 
+    # Fix issue #2786 if already tried to update to 3.18.1
+    modulemds_to_update = []
+    for modulemd in Modulemd.objects.filter(snippet__startswith="b\'"):
+        modulemd.snippet = eval(modulemd.snippet).decode("utf8")
```

Comment:
We definitely cannot `eval()` input that comes from the outside world - `eval()` executes the string as if it were code.